ts/src/yobit.ts

```diff
@@ -562,12 +561,14 @@ export default class yobit extends Exchange {
         } else {
             ids = this.marketIds (symbols);
         }
+        const idsLength = ids.join ('-');
         const idsString = ids.join ('-');
-        const maxLength = this.safeInteger (this.options, 'fetchTickersMaxLength', 2048);
+        const maxLength = this.safeInteger (this.options, 'maxUrlLength', 2048);
         // max URL length is 2048 symbols, including http schema, hostname, tld, etc...
         const lenghtOfBaseUrl = 30; // the url including api-base and endpoint dir is 30 chars
-        if (idsString.length + lenghtOfBaseUrl > maxLength) {
-            throw new ArgumentsRequired (this.id + ' fetchTickers() is being requested for ' + (idsString.length).toString () + ' markets, but it exceedes max URL length (' + maxLength.toString () + ' characters), please pass limisted symbols array to fetchTickers to fit in one request');
+        const actualLength = idsString.length + lenghtOfBaseUrl;
+        if (actualLength > maxLength) {
+            throw new ArgumentsRequired (this.id + ' fetchTickers() is being requested for ' + idsLength + ' markets (which has an URL length of ' + actualLength.toString () + ' characters), but it exceedes max URL length (' + maxLength.toString () + '), please pass limisted symbols array to fetchTickers to fit in one request');
```

Comment:
@ttodua First I think fetchTickers should work by default (even if we have to limit the number if markets sent) second, I don't believe this is a reasonable error message because it is excessively long and hard to read
![image](https://github.com/ccxt/ccxt/assets/43336371/14abc455-1bd3-41d0-b627-4df75a24947d)

![image](https://github.com/ccxt/ccxt/assets/43336371/93dc682b-19b7-401d-a568-13e721c45960)

Reply:
in second I agree, in first I might not :) because it would be direct misbehavior and issue of our code to randomly choose which symbols to fetch, as it will mislead/misguide users. not?

Reply:
@ttodua in that case, we can just make symbols mandatory to avoid this issue

Reply:
we just throw exception and letting users know to select their preferred symbols, so everyone will be on clear side. why should we arise extra confusions? let's user decide. 

but as you'll say.

Reply:
@ttodua in my opinion just does not make sense to have a method that does not work by default if in the end they have to select a number of symbols let be explicit about it then

Reply:
yeah, making symbols as mandatory sounds better

Reply:
pushed a commit, does that seem ok?

Reply:
ahh, I realized what you wrote now, in exception I accidentally included the full url. pushed an edit. now everything should be as you said.